packages/documentation/src/stories/internet-header/getting-started.stories.mdx

```diff
@@ -9,13 +9,22 @@ import { BADGE } from '@geometricpanda/storybook-addon-badges';
 
 The header for client-facing applications.
 
-- [Installation](#installation)
-- [Configuration](#configuration)
-- [Help](#help)
+<ul>
+  <li><a href="#installation" target="_self">Installation</a></li>
+  <li><a href="#configuration" target="_self">Configuration</a></li>
+  <li><a href="#help" target="_self">Help</a></li>
+</ul>
```

Comment:
There were redirecting to the iframe.html page. I'll do another to fix them across all doc. Storybook's issue: https://github.com/storybookjs/storybook/issues/15934